pkg/gofr/http/middleware/config.go

```diff
@@ -9,6 +9,8 @@ import (
 	"gofr.dev/pkg/gofr/config"
 )
 
+const LogDisableProbeKey = "LOG_DISABLE_PROBES"
```

Comment:
I would have used


```suggestion
const LogDisableProbeKey = "LOG_PROBES_DISABLED"
```

Or something like that. Maybe there are other env variables about disabling something